### ISSUE TITLE
Cannot install eclipse extras with the update site

### ISSUE THREAD

**Issue by konfri** [rherrmann/eclipse-extras#46]
Hey,

I really like you extras :) I was trying to update my Eclipse Oomph script (I was missing the new working set feature) but I'm somehow unable to install your bundles. Just to make sure that the guy in front of the keyboard isn't causing the issue with the oomph script I tried a regular install _"Help -> Install New Software..."_, but the same issue ... 

I made some screenshots using Eclipse Neon:

![eclipse_extra_install_01](https://cloud.githubusercontent.com/assets/16843990/16897525/438c390a-4bb4-11e6-9688-4526f00da332.png)
![eclipse_extra_install_02](https://cloud.githubusercontent.com/assets/16843990/16897527/4411a6a8-4bb4-11e6-86e5-6b90136d3c71.png)

Thanks for your plugins :)

Konstantin

**Comment by rherrmann** [comment]
Hi Konstantin, 

glad you like the plug-ins. I haven't seen this sort of error before.

From the window icon, it looks like you are using some Java EE package. I was able to install the extras into a fresh _Eclipse Neon for Java EE Developers_ as well as _Eclipse Mars.2 for Java EE Developers_ without problems. Hence I can only guess what is going wrong.

What exact version are you using? Did you try to install into a fresh Eclipse installation? Did you retry installing the plug-ins, maybe the _No repository found_ error is due to the repository being temporarily unavailable?

**Comment by konfri** [comment]
Hey :)

thank you for the amazingly fast reply. I use Eclipse Oomph to install all my eclipse instances and I guess some bundles got somehow corrupted. I deleted the folder that stores the jars and triggered a fresh installations (that downloaded all required bundles again). Now it works as expected :)

Later on, I discovered the _Bundle Pool Analysis_ features of Eclipse Oomph, that tells you which bundles are corrupt and repairs them :)

Thank you again and sorry for the confusion :)

Konstantin
